.eslintrc.js

```diff
@@ -41,6 +45,11 @@ module.exports = {
 		'react/jsx-sort-props': [ 'warn', { ignoreCase: true } ],
 		'react/no-deprecated': [ 'warn' ],
 		'react/sort-prop-types': [ 'warn', { ignoreCase: true } ],
+		'template-curly-spacing': [ 'warn' ],
+
+		// Deactivation - we should decide if we want them or not
+		'jest/valid-title': 0,
```

Comment:
@jlemay-godaddy will probably need your opinion on these

Reply:
I would go in favor of having one. Unless there is a scenario where titles don't make sense I'm not aware of?

Reply:
In that case, I think the errors are caused because we generate the test title from a variable.

Like this : https://github.com/godaddy-wordpress/coblocks/blob/master/src/blocks/services/test/save.spec.js#L16

Reply:
Oh yeah that must be it. Maybe we can exclude that rule on a case per case basis. If we do it across the board maybe then it's worth disabling.

Reply:
I mean, it's a title in a Jest file, I don't really have a strong opinion on that, it's only useful to quickly find the failing test. Let's enforce it with excludes if we have a few tests to fix, or disable the whole rule if it's too much to fix